components/sup/src/manager/service/spec.rs

```diff
@@ -192,21 +192,25 @@ impl IntoServiceSpec for protocol::ctl::SvcLoad {
         // per-service basis.
         base_spec.config_from = None;
 
-        let composite_binds = if let Some(ref list) = self.binds {
-            let binds: Vec<ServiceBind> = list.binds.clone().into_iter().map(Into::into).collect();
-            let (composite, _) = binds.into_iter().partition(|ref bind| bind.is_composite());
-            Some(composite)
-        } else {
-            None
+        // We permit the selective overriding of binds with CLI
+        // arguments; if any such overrides are present, this will
+        // pull them all out.
```

Comment:
This seems kind of contrary to the doc comment on L170:
```
     ///   (We do not set binds here, though, because that requires
     ///   specialized, service-specific handling.)
```